pkg/kubectl/resource_printer.go

```diff
@@ -1629,7 +1630,7 @@ func printEvent(event *api.Event, w io.Writer, options PrintOptions) error {
 
 // Sorts and prints the EventList in a human-friendly format.
 func printEventList(list *api.EventList, w io.Writer, options PrintOptions) error {
-	sort.Sort(SortableEvents(list.Items))
+	sort.Sort(events.SortableEvents(list.Items))
 	for i := range list.Items {
```

Comment:
It should be called. In your issue you don't get all of the events sorted because you probably have to deal with separate objects. Please move the discussion on the related issue since this is just a simple refactoring.